app/src/main/java/com/odysee/app/ui/channel/ChannelFragment.java

```diff
@@ -628,6 +632,14 @@ public void run() {
                             tabPager.setCurrentItem(3);
                         }
                     }, 500);
+                } else {
+                    // switch to the previously shown fragment (i.e. before navigating away and back)
+                    new Handler().postDelayed(new Runnable() {
```

Comment:
Try `new Handler(Looper.getMainLooper()).postDelayed` to avoid using a deprecated Handler constructor

Reply:
Thanks, done.